src/app/stream-deck/authorization/AuthorizationNotification.tsx

```diff
@@ -0,0 +1,43 @@
+import { t } from 'app/i18next-t';
+import { showNotification } from 'app/notifications/notifications';
+import { DeferredPromise } from 'app/stream-deck/util/deferred';
+import './AuthorizationNotification.scss';
+
+interface StreamDeckChallengeProps {
+  challenge: number;
+  onApprove?: () => void;
+  onCancel?: () => void;
+}
+
+function StreamDeckChallenge({ challenge, onApprove, onCancel }: StreamDeckChallengeProps) {
+  return (
+    <div className="stream-deck-authorization">
+      <span>{t('StreamDeck.Authorization.Title')}</span>
+      <div className="stream-deck-challenge">{challenge}</div>
+      <div className="stream-deck-authorization-buttons">
+        <button type="button" className="dim-button" onClick={onApprove}>
+          {t('StreamDeck.Authorization.Yes')}
+        </button>
+        <button type="button" className="dim-button" onClick={onCancel}>
+          {t('StreamDeck.Authorization.No')}
+        </button>
+      </div>
+    </div>
+  );
+}
+
+// Show notification asking for selection
+export function showStreamDeckAuthorizationNotification(
+  challenge: number,
+  promise: DeferredPromise,
+  onApprove?: () => void,
+  onCancel?: () => void
+) {
+  showNotification({
+    title: `Elgato Stream Deck`,
```

Comment:
Localize?

Reply:
mmm it's a product name, so not sure if we need to localize it (I already add the same string in the menu lang, but I believe that maybe is useless)

Reply:
Yeah I checked on their site and in every language it remains the same, so I remove the unnecessary string